diff --git a/cmake/nav2_package.cmake b/cmake/nav2_package.cmake
index 3f4977193..ad101b299 100644
--- a/cmake/nav2_package.cmake
+++ b/cmake/nav2_package.cmake
@@ -37,7 +37,7 @@ macro(nav2_package)
   endif()
 
   if(CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-    add_compile_options(-Wall -Wextra -Wpedantic -Werror -Wdeprecated -fPIC -Wshadow -Wnull-dereference)
+    add_compile_options(-Wall -Wextra -Wpedantic -fPIC)
     add_compile_options("$<$<COMPILE_LANGUAGE:CXX>:-Wnon-virtual-dtor>")
   endif()
 
